tools/templates/Makefile-common-rules.in

```diff
@@ -4,11 +4,13 @@
 
 spectest_checkout : @nfp(t/spec)@
 @nfp(t/spec)@:
-	git clone @roast_pull_url@ @nfp(t/spec)@
-	-cd @nfp(t/spec/)@ && git config remote.origin.pushurl @roast_push_url@
+	@echo(+++ Preparing the roast)@
+	$(NOECHO)git clone @roast_pull_url@ @nfp(t/spec)@
```

Comment:
I'm not sure that's an advantage. git clone can appear to hang, when one is on an unreliable network. Knowing that we're actually running git can help the user making the connection between "hang" and network.

Reply:
With `make -j` it could also appear somewhere in the middle of output if compilation is needed first. Not sure how much better is it.

Reply:
@niner It wasn't actually working the way you assumed, but I made it so because `make spectest` in a clean directory with two backends configured – it a total mess when `git` adds its progress reporting on top of it. But I took care of possible freezes with a new `monitor-process.pl` script which keeps track of a sub-process output and terminates with an error message if encounters a timeout.